### DISCUSSION TITLE
Stellarium, Ascom platform, Celestron driver- problems and questions

### DISCUSSION THREAD

**Discussion by MuenchenBeiNacht** [Stellarium/stellarium#1656] [Q&A]
Hello!

I hope you can help! So far I have controlled my Celestron Mount SE directly from Stellarium. It worked perfect by selecting the mode for telescopes directly and stored the correct COM interface inside stellarium. Then I was able to take over the coordinates of a object  / star in the telescope window of Stellarium and then clicked on "SCHWENKEN / SLEW" and the mount started moving and positioned correctly. For a variety of reasons, I have now also installed the Ascom platform including the corresponding Celestron drivers. Then I set the ASCOM mode in Stellarium and set the Celestron mount accordingly. When I start Stellarium now, a small Celestron window comes up in which I can slew and rotate the telescope by hand with four buttons. From Sharpcap I can now also slew and rotate the telescope. The position of the telescope and mount is correctly displayed in the Stellarium. Only if I now take over a position in the telescope window as usual and click on slew, nothing happens! Nada! No notification! No error message! And the mount doesn't move! If I move the telescope by hand with the small Celestron control window or inside Sharpcap, the telescope mark in the Stellarium display also moves accordingly. I can still slew to objects with the old telescope setting in direct mode in Stellarium as before, so basically nothing has changed. Only with ASCOM I cannot position from Stellarium! And it only works either ASCOM (in Sharpcap and not perfect in Stellarium) or the direct mode in Stellarium, but then it doesn't work with Shapcap.

Did I forget something during the ASCOM installation? What am I doing wrong? I look forward to every useful tip or tip.

Thank you - Kind regards!

**Comment by alex-w** [comment]
Do you use ASCOM 6.5?

**Reply by MuenchenBeiNacht** [reply]
Hello Alex,

Yes! Platform installation 6.5, service release 6.51! 

Brand new installation from original ASCOM web page. Drivers from Celestron web page. Here a screenshot.

![grafik](https://user-images.githubusercontent.com/72049618/117622668-2b417e80-b173-11eb-9ab5-3ceb43d48408.png)

Thank you for your support! Ewald

**Reply by alex-w** [reply]
ASCOM client in the Stellarium has error and it not work with version 6.5 of platform, sorry.

**Reply by MuenchenBeiNacht** [reply]
Hello Alex,
thanks for your answer. I am not 100% sure if I did understood correctly. 

Does it mean you ( and me ) found a problem within Stellarium? So I have to be patient until it will be fixed in Stellarium? 

Do you think it may be helpful to try to install a earlier / other version of ASCOM, in case I can find one? But there is no pressure from my site. I will be patient. If I can help in any way by testing or evaluating something, don't miss to ask, I will be open to help if necessary and when ever it is useful for you.

Or has your answer to be read, I did something wrong? For example, I mixed versions which could not work together? 

By the way Stellarium is such a great and fantastic application and all of the developer who help to maintain the software are great and doing a perfect job. Thanks to all members of the team who keep running this software.

Ewald
